configs/config.json.example

```diff
@@ -30,7 +30,7 @@
     "use_lucky_egg": false,
     "hatch_eggs": true,
     "longer_eggs_first": true,
-    "evolve_captured": false,
```

Comment:
What behavior should happen for people who haven't updated their config? If false won't work the same as "NONE" then I think we should have a check and if it is a boolean raise an exception to let users know the configuration has changed.